docs/otel-tracing.md

```
@@ -0,0 +1,284 @@
+# AWS OSIS (OpenSearch Ingestion Service)
+export OTEL_ENABLED=true
+export OTEL_EXPORTER_OTLP_ENDPOINT=https://pipeline-id.us-east-1.osis.amazonaws.com/otel-trace/v1/traces
+export OTEL_AWS_PROFILE=your-aws-profile
+export OTEL_AWS_REGION=us-east-1
+```
+
+## Span Hierarchy
+
+The tracing implementation creates a hierarchical span structure following Gen AI conventions:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)